packages/builder/src/components/backend/DataTable/ViewV2DataTable.svelte

```diff
@@ -28,6 +28,8 @@
     showAvatars={false}
     on:updatedatasource={handleGridViewUpdate}
     isCloud={$admin.cloud}
+    showReadonlyColumnsOptions
+    canSetReadonlyColumns={$licensing.isViewReadonlyColumnsEnabled}
```

Comment:
What's the purpose of the `showReadonlyColumnOptions` prop - just to control whether we show that readonly columns are a things only on views and not on other datasources? I would rather derive this from the datasource type, in the grid datasource store.

We do something similar for `hasBudibaseIdentifiers` in there.

Reply:
Done, I was not sure if it was better to have 2 props or having a more coupled grid